src/cosmos-ledger.ts

```diff
@@ -101,11 +101,17 @@ export default class Ledger {
         )
         transport = await TransportWebUSB.create(timeout * 1000)
       } catch (err) {
+        /* istanbul ignore next: specific error rewrite */
         if (err.message.trim().startsWith('No WebUSB interface found for your Ledger device')) {
           throw new Error(
             "Couldn't connect to a Ledger device. Please use Ledger Live to upgrade the Ledger firmware to version 1.5.5 or later."
```

Comment:
this is incorrect? should say 1.5.3 now...

Reply:
This talks about the FW not the app version